### PR TITLE
[Fix] Search bar labelling English/French

### LINKED ISSUE #945: Search bar on home page does not have label
**Describe the bug**
- The search bar on the home page does not have a label.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to the home page after login.
2. See Search bar.

**Expected behaviour**
- Need to add an appropriate label for the search bar that is English and French translateable.

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by SukhsimranpreetSekhon**
#### Changes introduced

- The search bar form input now has a label for English and French translation that will help for accessibility and screen reading.

#### Related issue(s)

fixes #945

#### Screenshots (if applicable)

English:
image

French:
image

#### Checklist
If the database has been modified:
- [ ] Update database diagram 
- [ ] Create new migration 

If API endpoints have been modified:
- [ ] Update backend documentation 
- [ ] Create/Update validators for the API endpoints 
- [ ] The API endpoints are properly secured with keycloak 

If the UI has been modified:
- [x] It is translated in both language (with no hardcoded text) 
- [ ] Has been tested on IE
- [x] The modifications are tab friendly
- [x] It is accessible

### CODE REVIEW

**Comment by dynamic11** [comment]
can you try rounding the search field using something similar to this:
https://stackoverflow.com/questions/42765528/how-to-make-rounded-raisedbutton-and-textfield

I'm not sure how it will look but lets talk about tomorrow quickly

**Comment by MenaMachado** [comment]
I agree with Ali's suggestion

File: services/frontend-v3/src/components/searchBar/SearchBarView.jsx [hunk 1]
```diff
@@ -58,7 +58,15 @@ const SearchBarView = ({
   // Generate the basic input field for basic search
   const getBasicField = () => {
     return (
-      <Form.Item style={{ width: "100%" }} label="" name="searchValue">
+      <Form.Item
+        style={{ width: "100%" }}
```
Comment (dynamic11) [inline services/frontend-v3/src/components/searchBar/SearchBarView.jsx]: can you move this to the css file

File: services/frontend-v3/src/components/searchBar/SearchBarView.jsx [hunk 2]
```diff
@@ -58,7 +58,15 @@ const SearchBarView = ({
...
+        label={
+          <text style={{ color: "white" }}>
```
Comment (dynamic11) [inline services/frontend-v3/src/components/searchBar/SearchBarView.jsx]: same here

**Review by dynamic11 — CHANGES_REQUESTED** [review]
pls see comments

File: services/frontend-v3/src/components/searchBar/SearchBarView.jsx (hunk 1 above)
Comment (SukhsimranpreetSekhon) [reply services/frontend-v3/src/components/searchBar/SearchBarView.jsx]: Fixed.

File: services/frontend-v3/src/components/searchBar/SearchBarView.jsx (hunk 2 above)
Comment (SukhsimranpreetSekhon) [reply services/frontend-v3/src/components/searchBar/SearchBarView.jsx]: Fixed.

**Comment by SukhsimranpreetSekhon** [comment]
> pls see comments

Fixed issues.

**Review by dynamic11 — APPROVED** [review]

**Pull request merged**